site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -11,7 +11,14 @@
 
 <div class="content">
     <header id="gradeable-info">
-        <h1>New submission for: {{ gradeable_name }}</h1>
+        <h1>
+            New submission for: {{ gradeable_name }}
+            {% if gradeable_url %}
```

Comment:
You should check if the url variable is defined in the current context 
by using `gradeable_url is defined` since its safer than to trust auto casting to a boolean - twig does some weird stuff there.

Reply:
This should probably have the same check as the other page that shows the gradeable_url:
https://github.com/Submitty/Submitty/blob/a41e73cb1ea163a9ab066178f85a3054a1609efb/site/app/templates/Navigation.twig#L56

so here would be:
```twig
{% if gradeable_url|trim != "" %}
```

Reply:
Yes, I'm on it!

Reply:
@shailpatels I thought Instructions for lab gradeable are only related to the `$ta_instructions ($gradeable->getTaInstructions())` but as per your suggestions, changes are introduced in lab gradeable, i.e both the links are shown (ta_instruction, instructions_url)..
![Screenshot 2020-01-22 at 6 02 21 PM](https://user-images.githubusercontent.com/48342617/72920511-5b2bff00-3d41-11ea-945d-deeffbaba5ce.png)

Let me know if something else needs to be changed, thanks!